entrypoint.py

```diff
@@ -5,7 +5,8 @@
 import query.accident_queries as accident_thunder_query
 import query.monthly_summary_ as monthly_summary_query
 import query.simple_station_ as simple_station_query
-
+import query.accident_queries as accident_query
```

Comment:
isn't this exactly the same as in line 5? Your optimized query with the input never runs?